test/integration/JobRoleIntegrationTest.ts

```diff
@@ -16,4 +16,18 @@ describe.skip('JobRoleIntegration', function () {
             return;
         }
     });
+
+    it('getJobRoleById should return job role details', async () => {
+        try {
+            const response: AxiosResponse = await axios.get(URL);  
+            
+            const data = response.data;
+            const status = response.status;
+
+            expect(status).to.equal(200);
+            expect(data).to.not.be.null;
+        } catch (e) {
+            return;
+        }
```

Comment:
what's this?

Reply:
Think I meant to come back to this and then forgot - oops! Changed this